### DISCUSSION TITLE
Allow to move individual comments to new thread or merge them into an existing thread

### DISCUSSION THREAD

**Discussion by Jackenmen** [community/community#52] [Discussions]
Sometimes there are situations in which a user comments in an existing thread and it would be better if that comment were made as a new thread or if it were made in a different thread.

I believe it's a rather common functionality in other forums, for example, vBulletin and I think also Discourse allow moderators to move posts to a new thread or merge them into an existing thread which will in turn split them off and leave a link in the original thread.

**Comment by becca** [comment]
👍 agreed! As we get closer to providing this functionality I might pipe in here with more questions. This is a feature we are looking to provide.

**Comment by andig** [comment]
Any update on this? Would be really great to have some basic moderation capabilities like split discussion, move issue (or issue comment) to (new) discussion etc.

**Comment by sskras** [comment]
ping @becca

**Comment by ARGAMX** [comment]
Any update on this? Would be really great to have some basic moderation capabilities like split discussion, move issue (or issue comment) to (new) discussion etc.
@becca
